### DISCUSSION TITLE
Is Birdnet-Pi Dead?

### DISCUSSION THREAD

**Discussion by kasgore** [mcguirepr89/BirdNET-Pi#1129] [General]
I have seen on a few sites where there are comments at Birdnet-Pi is dead and not maintained any longer. Is that true?

I have enjoyed the daylights out of this project more than any other project and have ran it for years now and it brings me a lot of satisfaction.

I know I just upgraded to a Raspi 4 and had to use some workarounds for the installer to work on the new version of Raspian Lite. That is what started me reading about the project and rather it is still under developement. 

If there is anything I can say to get the motivation going for the right people if they are taking a backseat to the project just let me know.

**Comment by Svardsten53** [comment]
#1003
